ui/components/component-library/button-base/button-base.scss

```diff
@@ -0,0 +1,79 @@
+.mm-button {
+  position: relative;
+  height: 40px;
+  padding: 0;
+  border-radius: 999px;
+  cursor: pointer;
+  color: var(--color-text-default);
+  background-color: #dcdcdc;
```

Comment:
should we be using a variable here as well?

Reply:
Nice catch @segun! We shouldn't have any static hex values in our code base if we can avoid it :)